.github/workflows/playwright.yml

```diff
@@ -0,0 +1,48 @@
+name: Playwright Tests
+
+on:
+  push:
+    branches: [next]
+  pull_request:
+
+jobs:
+  test:
+    name: Test component
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+    - name: Install dependencies
+      run: npm ci
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+    - name: Run Playwright tests
+      run: npm run test:component
+      env:
+        CI: 'true'
+    - name: Upload Playwright playwright report to GitHub Actions Artifacts
+      if: always()
```

Comment:
This line needed?

Reply:
I made it so that the report was always shown and during development it was clear whether new tests appeared or not

Reply:
I mean isnt's `if: always()` equal to not specifing `if` statement at all?